src/script/conversation/ConversationRepository.ts

```diff
@@ -1654,6 +1654,20 @@ export class ConversationRepository {
       if (conversationId) {
         await this.blacklistConversation(conversationId);
       }
+
+      /*
+        when we fetch this new MLS conversation
+        we get last_event_time equal to 1970-01-01T00:00:00.000Z as there
+        haven't been any events for this particular new conversation yet
+        since there are no previous proteus conversation there is no last_event_time
+        we can copy from so we set it to Date.now() to avoid
+        having a conversation with last_event_time equal to 1970-01-01T00:00:00.000Z
+        this is important because we use last_event_time to sort our list of conversations
+      */
+      if (mlsConversation.last_event_timestamp() < 10) {
+        mlsConversation.last_event_timestamp(Date.now());
+      }
```

Comment:
doesn't this guarantees that a conversation with no event will end up at the top of the list?
Do we know why a conv would appear on top if it's last timestamp is set to unix time zero?

Reply:
we sort conversations based on last timestamp, so the older the timestamp the lower it goes. this is for the case you don't have a previous proteus conversation to copy properties from and there is no event in the conversation. if we stick to setting system message time to new Date().toISOString(), we don't need this.

only visible issue with these timestamps is this:

<img width="161" alt="image" src="https://github.com/user-attachments/assets/06c2c34a-0b87-429d-bd8b-6c83a5725f09" />